packages/graphiql/src/components/DocExplorer/TypeLink.js

```diff
@@ -43,7 +43,7 @@ function renderType(type, onClick) {
     );
   }
   return (
-    <a className="type-name" onClick={event => onClick(type, event)}>
+    <a className="type-name" onClick={event => {event.preventDefault(); onClick(type, event)}} href="#">
```

Comment:
here, `event.preventDefault()` is added, whereas elsewhere it is removed, any particular reason?

Reply:
With the exception of form submission buttons, there’s no default behavior to prevent for `<button>` elements. For `<a>` elements, their default behavior is however to navigate, which we need to prevent if we want to use it as a sort of button. The `TypeLink` is actually a case where this is somewhat appropriate as it is used for internal links within the documentation explorer.

It is also worth pointing out that `<a>` elements need an `href` attribute for them to have any behavior. This has an effect on both keyboard operability and assistive technology as an `<a>` element without an `href` attribute is neither focusable nor communicates any semantics.

Reply:
Great explanation